src/users/components/TrustButton.vue

```diff
@@ -62,6 +62,18 @@
               width="20px"
             >
           </QBtn>
+          <QBtn
+            v-if="trusted && !user.isCurrentUser"
+            color="negative"
+            :loading="membership.trustUserStatus.pending"
```

Comment:
should this be `revokeTrustStatus`? (if that has been wired up...)

Reply:
Yes, fixed it. Now the loading spinner works.